bucket/stash.json

```
@@ -0,0 +1,27 @@
+{
+    "version": "0.15.0",
+    "description": "An organizer for your porn.",
+    "homepage": "https://stashapp.cc",
+    "license": "AGPL-3.0-only",
+    "suggest": {
+        "FFmpeg": [
+            "ffmpeg",
+            "ffmpeg-nightly"
+        ]
+    },
+    "url": "https://github.com/stashapp/stash/releases/download/v0.15.0/stash-win.exe",
+    "hash": "4a1dddce054fa73eeac692cdddf518ed8ce812b06fa05a11c00e6d74c5ba2739",
+    "bin": [
+        "stash-win.exe",
+        [
+            "stash-win.exe",
+            "stash"
+        ]
+    ],
```

```suggestion
    "pre_install": [
        "if (!(Test-Path \"$persist_dir\\config\")) {",
        "    New-Item \"$dir\\config\" -ItemType Directory | Out-Null",
        "}"
    ],
    "bin": [
        [
            "stash-win.exe",
            "stash-win",
            "-c \"$dir\\config\\config.yml\""
        ],
        [
            "stash-win.exe",
            "stash",
            "-c \"$dir\\config\\config.yml\""
        ]
    ],
    "persist": "config",
```

This will make the config file stored at `$dir\config`.